_source/user-guide/integrations/endpoints.md

```diff
@@ -39,3 +46,9 @@ Alerts are sent on ports 80 and 443 only.
 Setting a custom port as part of an endpoint URL
 will result in alerts not being sent to said endpoint.
 {:.info-box.important}
+
+* To test an endpoint after it is created, open it in edit mode. 
+Click **Run the test** and check the recepient for the test message. 
```

Comment:
```suggestion
Click **Run the test** and make sure the test message was received. 
```

"check the recipient" doesn't quite work here, but checking that it's received should do it.